ui/app/src/views/projects/ProjectTabs.tsx

```diff
@@ -0,0 +1,206 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Box, Stack, Tab, Tabs } from '@mui/material';
+import { ReactNode, SyntheticEvent, useCallback, useState } from 'react';
+import ViewDashboardIcon from 'mdi-material-ui/ViewDashboard';
+import VariableBoxIcon from 'mdi-material-ui/VariableBox';
```

Comment:
UX thought when pulling down the branch: I'm not crazy about the variables icon here:
<img width="433" alt="image" src="https://github.com/perses/perses/assets/9369625/79f53f04-bea1-4a66-a31f-8821b112b6e0">

Would using `mdi-material-ui/Variable` be better? I'm not 100% sold on it either but seems a little better:
<img width="425" alt="image" src="https://github.com/perses/perses/assets/9369625/5872ca88-3bd8-41b9-8b0a-f7bec5fb2a0b">